src/components/ims/ims-artifact.factory.js

```diff
@@ -264,39 +266,39 @@ export class ImsArtifactApi {
     if (entry) {
       return entry.description;
     } else {
-      return ;
```

Comment:
No. I did some changes then ran auto refactoring.  It fixed some row under my code. That is all.